crates/sui-json-rpc/src/transaction_builder_api.rs

```
@@ -50,17 +52,37 @@ impl AuthorityStateDataReader {
 
 #[async_trait]
 impl DataReader for AuthorityStateDataReader {
-    async fn get_objects_owned_by_address(
+    async fn get_owned_objects(
         &self,
         address: SuiAddress,
-    ) -> Result<Vec<SuiObjectInfo>, anyhow::Error> {
-        let refs: Vec<SuiObjectInfo> = self
-            .0
-            .get_owner_objects(address)?
-            .into_iter()
-            .map(SuiObjectInfo::from)
-            .collect();
-        Ok(refs)
+        options: Option<SuiObjectDataOptions>,
+        cursor: Option<ObjectID>,
+        limit: Option<usize>,
+        _checkpoint: Option<CheckpointId>,
```

let's throw an error if this is `Some`, since it's not supported at the moment